server/Cargo.toml

```diff
@@ -13,7 +13,8 @@ stackable-nifi-crd = { path = "../crd" }
 stackable-nifi-operator = { path = "../operator" }
 
 serde_yaml = "0.8"
-tokio = { version = "1", features = ["macros", "rt-multi-thread"] }
+tokio = { version = "1.6", features = ["macros", "rt-multi-thread"] }
```

Comment:
Comment does not refer to this line. In other PRs you added "the main binary" to the description in this toml.

Reply:
True....I'm inconsistent here. Spark and ZK don't have it.
I'm too lazy to change it again everywhere now :)